lib/promscrape/targetstatus.qtpl

```diff
@@ -103,6 +110,7 @@ function expand_all() {
 	"show_only_unhealthy": "true",
 	"endpoint_search": endpointSearch,
 	"label_search": labelSearch,
+	"active_tab": activeTab,
```

Comment:
Looks like activeTab isn't updated when actually navigating through the tabs.
This would switch to previous activeTab every time you press on the button

Reply:
it sets on request param when you click on tab

Reply:
I see. Try to switch to discovery tab and then press Unhealthy.
You'll be redirected to Targets tab. The opposite will happen if you reload page with Discovery tab active, then switch to Targets and press Unhealthy - you'll be taken to Discovery tab back again.

Reply:
logic updated, my fault, forget update handlers